deploy/kubernetes/fluentd-sumologic.yaml

```diff
@@ -378,11 +378,51 @@ data:
     <source>
       @type events
     </source>
+    <source>
```

Comment:
this will only add metrics to fluentd-events, but we probably want to instrument fluentd deployment for logs/metrics?  Not sure how valuable it is to get fluentbit metrics.

Reply:
Yes.  Totaly going to, I need to play with it more.  I was hitting an issue getting it to work with our plugin.  The Prometheus fluentD plugin that is used to instrument was not happy with the fluentD deployment.  Fluent-Bit metrics could be used to identify issues with dropped data as it will tell us what it is reading by tag, which is the full file path.